cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/query_enhancements/a_check.spec.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+import { WORKSPACE_NAME, DATASOURCE_NAME } from './constants';
+import { SECONDARY_ENGINE } from '../../../../../utils/constants';
+
+const miscUtils = new MiscUtils(cy);
+
+describe('Workspace Commands / API Test', () => {
+  before(() => {
+    // Load test data
+    cy.setupTestData(
+      [
+        'cypress/fixtures/query_enhancements/data-logs-1/data_logs_small_time_1.mapping.json',
```

Comment:
if we refactor the structure of the fixtures to not care about the plugin for example `cypress/fixtures/{file name}`

we could make `setupTestData` just automatically set the path so that all we have to do is pass the file name for example

`setupTestData('small_time_range_1.mapping.json')`

then in the setupTestData we automatically just preprend `cypress/fixtures/` to it. this would make it easier and less prone to error if the file path is too long